Add debugging a failed Fusion run section to run visibility (#8914)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/run-visibility.md

<Lightbox src="/img/docs/dbt-cloud/deployment/fusion-logs.png" width="80%" title="Structured logs in Fusion" />

#### Debugging a failed Fusion run

> When a job on the <Constant name="fusion_engine" /> fails, the run details page displays a **Debug on Fusion** banner. Click **Debug on Fusion** to open the <Constant name="studio_ide" /> with Fusion enabled, so you can reproduce and investigate the failure interactively.

Author: ```suggestion
When a job on the <Constant name="fusion_engine" /> fails, the run details page displays a This run failed on Fusion banner. You can click **Debug this in Studio** and select **Debug on Fusion** to open the <Constant name="studio_ide" /> with Fusion enabled, so you can reproduce and investigate the failure interactively.
```

> Before opening the <Constant name="studio_ide" />, dbt automatically sets your `DBT_DEVELOP_CORE_VERSION` environment variable to `latest-fusion`. The button is disabled while this setting is being applied and shows a _Preparing Fusion debug..._ state. If the override fails to save, an inline error message is displayed.

Author: ```suggestion
Before opening the <Constant name="studio_ide" />, dbt sets your environment variable to `latest-fusion` so you can debug using Fusion. 
```
